### ISSUE TITLE
Mouse wheel scrolling problems only on termite

### ISSUE THREAD

**Issue by mindstormer12** [thestinger/termite#240]
When using termite, I have problems using the mouse wheel to scroll up and down the terminal. Even when incrementing the mouse wheel upward or downward by one notch slowly, the cursor may go the opposite direction and may not even skip the default number of lines. For example, it is near impossible to scroll from the top of the terminal to the bottom of the terminal when looking at previous commands. I removed my config file and used the default one but the problem still persists.

 I have no issues regarding mouse wheel scrolling on other terminals whatsoever. I use the Logitech G9 mouse out of the box without any additional configurations and am i3 window manager. Using the arrow keys to scroll is fine.

**Comment by thestinger** [comment]
Does this happen in other VTE3 terminals, such as the `vte-2.91` application shipping with the library?

**Comment by mindstormer12** [comment]
No, just launched vte-2.91 and mouse wheel scrolling is working flawlessly, so just termite. I'm new to Linux and github and it's really cool to see such fast response.

**Comment by mindstormer12** [comment]
Apparently installing gpm fixed the issue (I hadn't installed gpm yet--I just followed the wiki entries on Arch installation and Termite). I then did pacman -Rns to remove gpm to see if the problem exists without gpm, but it still fixed even after a complete removal of gpm and a reboot. 

Out of curiosity, would this still be considered a bug (since vte-2.91 worked without gpm) or can it be closed? Perhaps it would be better for the termite entry in the Arch Wiki to recommend users to install gpm or is this implicit?
